HiddenWallet.Gui/app/js/status-client.js

```diff
@@ -0,0 +1,255 @@
+var mempool = 0;
+var trackerHeight = 0;
+var headerHeight = 0;
+var blocksLeft = "";
+var nodeCount = 0;
+var walletState = "NotStarted";
+var torState = "NotStarted";
+var progressType = "success";
+var progress = "10";
+var connection;
+var mixerStatusResult;
+var isTumblerOnline = false;
+
+function signalrStatusUpdate() {
+    connection = new signalR.HubConnection('http://localhost:37120/daemonHub');
```

Comment:
The code fails here. Maybe I ruined something on the back end so it won't progress further from here?